misc/extension_api_validation/4.2-stable.expected

```diff
@@ -73,3 +73,20 @@ GH-87668
 Validate extension JSON: Error: Field 'classes/Font/methods/find_variation/arguments': size changed value in new API, from 8 to 9.
 
 Added optional "baseline_offset" argument. Compatibility method registered.
+
+
+GH-87888
+--------
+Validate extension JSON: API was removed: classes/BoneAttachment3D/methods/get_external_skeleton
+Validate extension JSON: API was removed: classes/BoneAttachment3D/methods/set_external_skeleton
+Validate extension JSON: API was removed: classes/BoneAttachment3D/properties/external_skeleton
+Validate extension JSON: API was removed: classes/BoneAttachment3D/methods/get_use_external_skeleton
+Validate extension JSON: API was removed: classes/BoneAttachment3D/methods/set_use_external_skeleton
+Validate extension JSON: API was removed: classes/BoneAttachment3D/properties/use_external_skeleton
+Validate extension JSON: API was removed: classes/OpenXRHand/methods/get_hand_skeleton
+Validate extension JSON: API was removed: classes/OpenXRHand/methods/set_hand_skeleton
+Validate extension JSON: API was removed: classes/OpenXRHand/properties/hand_skeleton
+Validate extension JSON: API was removed: classes/Skeleton3D/properties/animate_physical_bones
+
+These definitions have been moved to those base classes, so the APIs are actually available or compatibility methods registered.
+Some properties were renamed for integration, but the old setter/getters are kept.
```

Comment:
Two of the warnings seem to no longer occur, according to this lint?

![grafik](https://github.com/godotengine/godot/assets/708488/52e9ae9c-40ef-4fa0-943e-8cdfc3109a47)